components/floatinglabel/validation.md

```diff
@@ -0,0 +1,83 @@
+---
+title: Validation
+page_title: FloatingLabel Validation
+description: The FloatingLabel can change its styling when it is inside a validated form and the associated form field is invalid. How to integrate the Blazor floating label with form validation.
+slug: floatinglabel-validation
+tags: telerik,blazor,floatinglabel,floating,label,form,validation
+published: True
+position: 10
+---
+
+# Floating Label Validation
+
+The Blazor FloatingLabel integrates with form validation. When a form field is invalid, the floating label will change color to suggest user action.
```

Comment:
```suggestion
The Blazor FloatingLabel integrates with form validation of compatible Telerik components. 
```
Although it is specified in the overview article, it might still be useful to have 3 more words here, so that there is less risk customers imply that this will work with any form component.